docs/website/docusaurus.config.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const theme = require('prism-react-renderer/themes/github');
+
+module.exports = {
+  title: 'Renovate Docs',
+  tagline: 'Documentation for Renovate',
+  url: 'https://docs.renovate.com',
+  baseUrl: '/',
+  onBrokenLinks: 'warn',
+  onDuplicateRoutes: 'throw',
+  favicon: 'img/logo.png',
+  themeConfig: {
+    colorMode: {
+      disableSwitch: true,
+      respectPrefersColorScheme: true,
+    },
+    navbar: {
+      hideOnScroll: true,
+      title: 'Renovate Docs',
+      logo: {
+        alt: 'Renovate Logo',
+        src: 'img/logo.png',
+      },
+      items: [
+        {
+          href: 'https://github.com/renovatebot/renovate',
+          label: 'GitHub',
+          position: 'right',
+        },
+      ],
+    },
+    footer: {
+      style: 'dark',
+      links: [],
+      logo: {
+        alt: 'Renovate Logo',
+        src: '/img/logo.png',
+      },
```

Comment:
The Renovate logo is way too big, causing the footer to be really high.
I think we need to find a way to scale down the Renovate logo.